frontend/src/static/js/components/webstatus-overview-filters.ts

```diff
@@ -240,23 +413,14 @@ export class WebstatusOverviewFilters extends LitElement {
   renderExportButton(): TemplateResult {
     const exportToCSV = () => {
       this.exportDataStatus = TaskStatus.PENDING;
-
-      // dispatch an event via CustomEvent
-      const event = new CustomEvent('exportToCSV', {
-        bubbles: true,
-        composed: true,
-        cancelable: true,
-        detail: {
-          callback: () => {
-            this.exportDataStatus = TaskStatus.COMPLETE;
-          },
-        },
+      this.exportToCSV(() => {
+        this.exportDataStatus = TaskStatus.COMPLETE;
       });
-      this.dispatchEvent(event);
     };
 
     return html`
       <sl-button
+        id="export-to-CSV-button"
```

Comment:
```suggestion
        id="export-to-csv-button"
```

attribute values should be lowercase https://google.github.io/styleguide/htmlcssguide.html#Capitalization